integration-tests/no-awt/pom.xml

```diff
@@ -90,27 +90,4 @@
         </plugins>
     </build>
```

Comment:
@gsmet Hmm, would the `quarkus-integration-test-no-awt` keep running in native? The point is to run it in native mode only, no JVM mode tests. 
It tests that the correct error is logged for users that do something with ImageIO/AWT without having AWT extension installed.

Reply:
Yes. The native tests are run with a profile coming from the parent.